datalake/src/main/java/com/sequenceiq/datalake/service/SdxEventsService.java

```diff
@@ -117,6 +118,10 @@
     }
 
     private List<CDPStructuredEvent> retrievePagedCloudbreakServiceEvents(SdxCluster sdxCluster, Integer page, Integer size) {
+        if (sdxCluster.getDeleted() != null) {
+            return Collections.emptyList();
+        }
+
```

Comment:
We are intentionally getting the events for deleted datalake as well. When the datalake is resized, we also need to show the events generated for the data lake deleted.

What is the motivation for this change?

Reply:
Getting Cloudbreak events from deleted datalakes literally does not work. This will throw an error and show nothing on the UI. Events from the SDX service are still shown.
I tested this specifically with a resize case and made sure the correct events are shown.